pkg/minishift/addon/command/cat_command_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"io/ioutil"
+	"testing"
+
+	pkgTesting "github.com/minishift/minishift/pkg/testing"
+	"github.com/stretchr/testify/assert"
+	"os"
+)
+
+func Test_cat_command_when_file_exist(t *testing.T) {
+
+	// Creating a tmp file with test content.
+	content := []byte("This file to used to test the cat command output.")
+	tmpfile, err := ioutil.TempFile("", "test.txt")
+	if err != nil {
```

Comment:
We can remove the `if err !=nil` checks here, as `assert.NoError` already does that check.